KMNavigationBarTransition/UINavigationController+KMNavigationBarTransition.m

```diff
@@ -49,6 +49,10 @@ + (void)load {
     });
 }
 
+- (UIColor *)km_containerViewBackgroundColor {
```

Comment:
Sorry, I don't understand why you need to add this method, you can change the color of the container view in `viewWillLayoutSubviews` method if you want.